providers/implementations/keymgmt/ml_kem_kmgmt.c.in

```diff
@@ -460,6 +460,57 @@ static int ml_kem_key_fromdata(ML_KEM_KEY *key,
     return ossl_ml_kem_parse_public_key(pubenc, publen, key);
 }
 
+#ifndef FIPS_MODULE
+static int ml_kem_refetch_digests(ML_KEM_KEY *key, const OSSL_PARAM params[])
+{
+    const OSSL_PARAM *p;
+    const char *properties = NULL;
+    EVP_MD *tmp = NULL;
+
+    p = OSSL_PARAM_locate_const(params, OSSL_ALG_PARAM_PROPERTIES);
```

Comment:
Adding a new parameter should also update the importable list which you haven't.  In this case it also means splitting the list into separate import and export lists and having separate functions for each.

I'd also prefer to not calll the locate function and use the generated decoder which might mean pulling this inside of `ml_kem_key_fromdata` which does the decode currently.

Reply:
Do I correctly understand that if we add these parameters also on export, it will let us keep import/export in sync?